bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -414,13 +414,14 @@ public Closeable createClosable(final Channel channel) {
           public void close() throws IOException {
             ChannelImpl channelImpl = (ChannelImpl) channel;
             channelImpl.shutdown();
-            while (!channelImpl.isTerminated()) {
-              try {
-                channelImpl.awaitTermination(CHANNEL_TERMINATE_WAIT_MS, TimeUnit.MILLISECONDS);
-              } catch (InterruptedException e) {
-                Thread.interrupted();
-                throw new IOException("Interrupted while sleeping for close", e);
-              }
+            try {
+              channelImpl.awaitTermination(CHANNEL_TERMINATE_WAIT_MS, TimeUnit.MILLISECONDS);
```

Comment:
n tries isn't all that helpful.  awaitTermination waits until the channel is terminated.  Any configuration ought to be in seconds rather than tries.  Tries really isn't all that meaningful in this case.